packages/camera/camera_windows/windows/camera_plugin.cpp

```diff
@@ -341,6 +363,58 @@ void CameraPlugin::StopVideoRecording(
   }
 }
 
+void CameraPlugin::StartImageStream(
+    int64_t camera_id,
+    std::function<void(std::optional<FlutterError> reply)> result) {
+  // check if request already exists
+  Camera* camera = GetCameraByCameraId(camera_id);
+  if (!camera) {
+    return result(FlutterError("camera_error", "Camera not created"));
+  }
+  if (camera->HasPendingResultByType(PendingResultType::kStartStream)) {
+    return result(
+        FlutterError("camera_error", "Pending start stream request exists"));
+  }
+
+  if (!event_sink) {
+    return result(FlutterError("camera_error",
+                               "Unable to make event channel from windows"));
+  }
+
+  if (camera->AddPendingVoidResult(PendingResultType::kStartStream,
+                                   std::move(result))) {
+    CaptureController* cc = camera->GetCaptureController();
+    assert(cc);
+    cc->StartImageStream(std::move(event_sink));
+  }
+}
+
+void CameraPlugin::StopImageStream(
+    int64_t camera_id,
+    std::function<void(std::optional<FlutterError> reply)> result) {
+  // check if request already exists
+  Camera* camera = GetCameraByCameraId(camera_id);
+  if (!camera) {
+    return result(FlutterError("camera_error", "Camera not created"));
+  }
+  if (camera->HasPendingResultByType(PendingResultType::kStartStream)) {
+    return result(
+        FlutterError("camera_error", "Pending start stream request exists"));
+  }
```

Comment:
Should this be `PendingREsultType::kStopStream`?

```suggestion
  }
  if (camera->HasPendingResultByType(PendingResultType::kStopStream)) {
    return result(
        FlutterError("camera_error", "Pending stop stream request exists"));
  }
```